src/common/utils.js

```diff
@@ -79,3 +79,16 @@ export class AsyncActionType {
     return `${this.topic}__${this.name}__RESET`;
   }
 }
+
+export function getDateWithDaysOffset(days) {
+  const date = new Date();
+  date.setUTCDate(date.getUTCDate()-days);
+  return date;
+}
+
+export function getISODateString(date) {
+  const isoDate = date.toISOString();
+  // we only want the date, not the time
+  const isoDateParts = isoDate.split('T')
+  return isoDateParts[0]
```

Comment:
I had originally tried something along the lines of:

```
  const day = date.getUTCDate()
  // getUTCMonth is 0-indexed
  const month =  date.getUTCMonth() + 1;
  const year = date.getUTCFullYear();

  return `${year}-${month}-${day}`;
```
however, month is zero indexed in Javascript's Date class, which is annoying, and formatting the ISO string myself was more difficult than I expected (e.g. I had trouble getting a leading zero to show for dates 1-9). I have decided on this approach instead. `toISOString`is based on the standard ISO 8601 simplified format, so I feel comfortable splitting on the `T` character.